stdlib/ballerina-http/src/main/ballerina/http/http_secure_client.bal

```diff
@@ -21,12 +21,22 @@ import ballerina/mime;
 
 @final string EMPTY_STRING = "";
 @final string WHITE_SPACE = " ";
+@final string COLON = ":";
 @final string CONTENT_TYPE_HEADER = "Content-Type";
 @final string BASIC_SCHEME = "basic";
 @final string OAUTH_SCHEME = "oauth";
 @final string JWT_SCHEME = "jwt";
 
 @Description {value:"An HTTP secure client for interacting with an HTTP server with authentication."}
+documentation {
+    Provides the HTTP actions for interacting with an HTTP server. Apart from the standard HTTP methods, `forward()`
+    and `execute()` functions are provided. `forward()` takes an incoming HTTP requests and sends it to an upstream
```

Comment:
Remove the 3rd sentence

Reply:
Also, instead of copying the description for the generic HTTP client verbatim, can you include something about how this client provides security?